src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -62,6 +79,46 @@ void PocoHTTPClient::MakeRequestInternal(
     auto uri = request.GetUri().GetURIString();
     LOG_DEBUG(log, "Make request to: {}", uri);
 
+    enum class ES3MetricType
+    {
+        Microseconds,
+        Count,
+        Errors,
+        Throttling,
+        Redirects,
+
+        EnumSize,
+    };
+
+    auto SelectMetric = [&request](ES3MetricType type)
```

Comment:
Function names should begin with lowercase letter (style).

Reply:
Fixed